### PR TITLE
xorg/libs: Continue packaging required x11 libs

### PR DESCRIPTION
**Pull request by joebonrichie**
Initially included for qt5/gtk3

### CODE REVIEW

**Comment by joebonrichie** [comment]
summit emul32 deps are currently borked. Be weary of libxi/libxtst build order.

**Comment by joebonrichie** [comment]
emul32 bug has been washed out now. Depends on https://github.com/snekpit/main/pull/63.

File: xorg/libs/libxinerama/monitoring.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+releases:
+    id: 1779
+security: {}
```
Comment (ReillyBrogan) [inline xorg/libs/libxinerama/monitoring.yml:3]:
```
security:
  cpe:
  - vendor: x
    product: libxinerama
```

**Comment by ReillyBrogan** [comment]
At least one of these is missing the CPE. I think what might be happening is that you are checking the product with the mixed capitalization original name (`libXinerama`). You should use all-lowercase instead (`libxinerama`). 

I'm just going to go ahead and add the CPEs after merging this so as to not slow you down.

**Pull request merged**